util/util.go

```diff
@@ -68,6 +68,7 @@ type CommandSpec struct {
 	Docker      *Docker  `json:"docker,omitempty" yaml:"docker,omitempty"`
 	Binary      *Binary  `json:"binary,omitempty" yaml:"binary,omitempty"`
 	PipelineID  int      `json:"pipelineId,omitempty" yaml:"pipelineId,omitempty"`
+	SpecPath    string   `json:"-" yaml:"-"`
```

Comment:
Is it better to use name of `YamlPath` or `SpecYamlPath`, because [this parsePublishCommand](https://github.com/screwdriver-cd/sd-cmd/blob/master/publisher/publisher.go#L71) return yamlPath?
If you think SpecPath is better, you do not need fix this name. This is little comment.

Reply:
`Publisher` struct has `specPath` field...
We should use `SpecYamlPath`. Thanks.